doc/classes/CheckBox.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" ?>
-<class name="CheckBox" inherits="Button" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+<class name="CheckBox" inherits="Button" keywords="radio button" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
```

Comment:
I think "button" is redundant here, "radio" should suffice.

https://github.com/godotengine/godot/pull/105649 also adds `radio` as a keyword to ButtonGroup.